### DISCUSSION TITLE
Translation to Spanish

### DISCUSSION THREAD

**Discussion by paladique** [microsoft/Data-Science-For-Beginners#162] [General]
This is the discussion thread for those who are helping with this translation to divide up the work, as well as review. This is Issue #132

**Comment by OscarRG** [comment]
@paladique Thanks for opening the discussion!

As we can see, there is quite a lot to translate. I would say that we could divide it in different issues formed by the different sections:

- Introduction base README
- Working with Data base README
- Vizualization base README
- Data Science Lifecycle base README 
- Data Science in the Cloud base README
- Data Science in the Wild base README.

I would like to start with: `1-Introduction/README.md` & `1-Introduction/01-defining-data-science/README.md`.

What do you think about it? :)

**Reply by Carababusha** [reply]
Hi @paladique! I'd love to help with any Spanish translations that aren't assigned yet. Has all work been completed?

**Reply by hereje** [reply]
Well, I made the translations for base readme file on module 1: sections 1 to 3.

#173

**Reply by Carababusha** [reply]
Hi @hereje! I'm trying to find my way around the project. Lots of ES files all over the place but no finished translations that I can see (like [here](https://github.com/microsoft/Data-Science-For-Beginners/tree/main/1-Introduction/03-defining-data/translations), for example). I just don't want to step on anyone's toes if someone is working on those files already! 🙈

**Reply by OscarRG** [reply]
Hello @hereje Please dont start a PR without it being assigned to you or please ask if X file is already being translated. As you can see, there was already interest in translating `1-Introduction/01-defining-data-science/README.md`.

**Reply by hereje** [reply]
OK, Understood.
 @OscarRG Could you please assign me the translation for `1-Introduction/03-defining-data/README.md`?

**Reply by OscarRG** [reply]
Hi @hereje, I will edit this discussion with all the files to translate so that we can organize it. In the meantime yes, you can translate the file `1-Introduction/03-defining-data/README.md` :)

**Reply by hereje** [reply]
Done, See #173

**Reply by hereje** [reply]
Hello guys.

If you don't mind. I'll translate the following files -one per PR-

* 6-Data-Science-In-Wild/README.md
* 6-Data-Science-In-Wild/20-Real-World-Examples/assignment.md
* 6-Data-Science-In-Wild/20-Real-World-Examples/README.md

Are you OK with this?

Please let me know, so that I can start the translation.

**Reply by alondraSanchezM** [reply]
Hi team. 

I would like to start helping in the translation with these files:

- Visualization base README
  -  Quantities README
  -  Quantities assignment

I hope there is no problem; anything please let me know(:

**Reply by alondraSanchezM** [reply]
Hi all! I will continue translating the Visualization section.

**Reply by Carlosbogo** [reply]
Hi all! I'm interested in helping with the translation.

If possible I would like to start translating from some of the first topics covered in the curriculum, because that way it's easier for me to know what has already been covered and what hasn't in order to make explanations more understandable and less repetitive, but I haven't been able to find a list of topics that need to be translated (or reviewed) that's up to date. Is there anything like that yet?

On another note, are there any preferences on the accent used in the translation? (I mean if it should be done using expressions from Spain or from Mexico or from any other spanish speaking country).

**Reply by Carlosbogo** [reply]
For now I'll be working on the first few lessons. If there are any problems feel free to contact me.
